crypto/fipsmodule/ec/ec_nistp.h

```diff
@@ -34,7 +34,7 @@ typedef uint64_t ec_nistp_felem_limb;
 typedef uint32_t ec_nistp_felem_limb;
 #endif
 
-// ec_nistp_felem_meth is a struct that holds pointers to implementations of field
+// ec_nistp_meth is a struct that holds pointers to implementations of field
```

Comment:
The rest of the documentation would need to be adjusted to not only mention field arithmetic functions.